bench/bench-vptree.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var cities = require('all-the-cities');
+var VPTreeFactory = require('vptree');
+
+
+var earthRadius = 6371;
+var rad = Math.PI / 180;
+function greatCircleDist(lng, lat, lng2, lat2, cosLat, sinLat) {
+    var cosLngDelta = Math.cos((lng2 - lng) * rad);
+    return earthRadius * Math.acos(greatCircleDistPart(lat2, cosLat, sinLat, cosLngDelta));
+}
+function greatCircleDistPart(lat, cosLat, sinLat, cosLngDelta) {
+    var d = sinLat * Math.sin(lat * rad) +
+            cosLat * Math.cos(lat * rad) * cosLngDelta;
+    return Math.min(d, 1);
+}
+function distance(p1, p2) {
+    var lng = p1.lon, lat = p1.lat, lng2 = p2.lon, lat2 = p2.lat;
+    return greatCircleDist(lng, lat, lng2, lat2, Math.cos(lat * rad), Math.sin(lat * rad));
+}
```

Comment:
Let's fold that into one function like here since the breakdown is unnecessary in this case https://github.com/mourner/geokdbush/blob/master/bench/bench-naive.js#L43